osu.Game/Rulesets/Mods/ModTimeRamp.cs

```diff
@@ -26,6 +26,8 @@ public abstract class ModTimeRamp : Mod, IUpdatableByPlayfield, IApplicableToBea
         [SettingSource("Final rate", "The final speed to ramp to")]
         public abstract BindableNumber<double> FinalRate { get; }
 
+        public override string IconTooltip => $"{Name} ({InitialRate.Value}x to {FinalRate.Value}x)";
```

Comment:
I feel like the mod name and the brackets shouldn't be included in this override to make sure display is standardised.